qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -122,44 +130,52 @@ def _default_experiment_options(cls) -> Options:
 
         return options
 
-    def __init__(self, qubit: int):
-        """Setup a fine amplitude experiment on the given qubit.
-
-        Args:
-            qubit: The qubit on which to run the fine amplitude calibration experiment.
+    @classmethod
+    def _default_calibration_options(cls) -> Options:
+        """Default calibration options for the experiment.
+
+        Calibration Options:
+            schedule_name (str): The name of the schedule to retrieve from the Calibrations,
```

Comment:
These look like they could be only init args instead of options

Reply:
See here: 5c2d9c1